demo/src/app/pages/modules/datepicker/datepicker.page.ts

```diff
@@ -95,6 +95,11 @@ export class DatepickerPage {
                     defaultValue: "datetime"
                 },
                 {
+                    name: "pickerInitialDate",
+                    type: "Date",
+                    description: "Sets the intial date to display (null = today)."
```

Comment:
Change to `Sets the intial date to display when no date is selected.`. Add `new Date()` as the `defaultValue`